modules/pam_faillock/main.c

```diff
@@ -123,8 +126,80 @@ args_parse(int argc, char **argv, struct options *opts)
 static void
 usage(const char *progname)
 {
-	fprintf(stderr, _("Usage: %s [--dir /path/to/tally-directory] [--user username] [--reset]\n"),
-		progname);
+	fprintf(stderr,
+		_("Usage: %s [--dir /path/to/tally-directory] "
+		  " [--user username] [--reset] [--legacy-output]\n"), progname);
+
+}
+
+static int get_local_time(time_t when, char *timebuf, unsigned long timebuf_size)
```

Comment:
Please consider using `size_t` instead.

Also note that at least in this file the type of function is placed on a separate line.

Reply:
Yeah, missed it. Will do.